pkg/apis/samplesoperator/v1alpha1/zz_generated.deepcopy.go

```diff
@@ -13,8 +13,8 @@ func (in *SamplesResource) DeepCopyInto(out *SamplesResource) {
 	*out = *in
 	out.TypeMeta = in.TypeMeta
 	in.ObjectMeta.DeepCopyInto(&out.ObjectMeta)
-	out.Spec = in.Spec
-	out.Status = in.Status
+	in.Spec.DeepCopyInto(&out.Spec)
+	in.Status.DeepCopyInto(&out.Status)
```

Comment:
@gabemontero i'm not sure why this is changing...  i just did a full generate, so either:

1) you haven't done it in a while and i have a newer sdk-generator than you
2) my sdk-generator is newer than yours

Reply:
could be either or both ... I have not done any changes in the vendor or generated code spaces since initially seeding the repo for SDK and getting the vendoring of client-go sorted out with that whole `glide` vs. `dep` foray.